### ISSUE TITLE
Naming the evaluation/metrics module

### ISSUE THREAD

**Issue by jni** [scikit-image/scikit-image#4041]
## Description

See #4025 for the start of this discussion.

We have just added a module that contains quality evaluation functions, including functions to evaluate segmentation quality, such as adaptive rand index and variation of information, as well as image similarity measurements, such as mean squared error and structural similarity, and a denoising quality measurement in PSNR. (The latter three already existed in the library in some form, but imho were misclassified.)

Thanks @seanbudd, @leGIT-bot and @rbavery for your work on that module!

Before we release 0.16, we should make sure we are happy with the API. Some, including @stefanv, suggested that "evaluate" or "evaluation" doesn't quite capture what these functions do, and that "metrics" might be a better name (and this is how it has been merged in the end). Rather than have a long, drawn-out discussion that prevented the functionality from being merged, I suggested that we merge and discuss how to rename (or whether to rename) here.

**Comment by stefanv** [comment]
`metrics` works for me; no objections.  Happy to hear other suggestions too.

**Comment by sciunto** [comment]
> We have just added a module that contains quality evaluation functions, including functions to evaluate segmentation quality, such as adaptive rand index and variation of information, as well as image similarity measurements, such as mean squared error and structural similarity, and a denoising quality measurement in PSNR. (The latter three already existed in the library in some form, but imho were misclassified.)
> 

Challenge completed, you didn't use the word metric! :laughing:

Argument 1: The reason I prefered "metrics" to "evaluation" is that I asked myself, evaluation of what?

Argument 2: I like metrics because scikit-learn use the same vocabulary for something similar: https://scikit-learn.org/stable/modules/classes.html#module-sklearn.metrics

**Comment by JDWarner** [comment]
I'm fine with `metrics`.

**Comment by grlee77** [comment]
PSNR and SSIM are often referred to as image quality assessment (IQA) methods, so perhaps `assessment` could be an option. 

I am fine with `metrics` and think it is easier to type, though.

**Comment by alexandrejaguar** [comment]
I was sure I commented previously, but what the heck. I was into `evaluate`, but I am +1 on `metrics` too.

**Comment by jni** [comment]
Ok, I think the consensus is about as strong as can be on `metrics`, which is great because it is the option requiring the least amount of work. =) Thanks all for chiming in!
